pkg/deployments/tasks/20220704-timelock-authorizer/input/mainnet.ts

```diff
@@ -0,0 +1,250 @@
+  ]),
+  createRoleData(DAO_MULTISIG, SmartWalletChecker.output().SmartWalletChecker, [
+    SmartWalletChecker.actionId('SmartWalletChecker', 'denylistAddress(address)'),
+    SmartWalletChecker.actionId('SmartWalletChecker', 'allowlistAddress(address)'),
+  ]),
+  createRoleData(BLABS_OPS_MULTISIG, EVERYWHERE, [
+    // This permission grants powers to call `checkpoint()` on all of SingleRecipientGauges, PolygonRootGauges, ArbitrumRootGauges.
+    SingleRecipientGauge.actionId('SingleRecipientGauge', 'checkpoint()'),
+  ]),
+  // BALTokenHolder.withdrawFunds(address, uint256) (veBAL BALTokenHolder)
+  // Note this actionId can't be pulled from the json file as the BALTokenHolder is not listed there.
+  {
+    role: '0x79922681fd17c90b4f3409d605f5b059ffcbcef7b5440321ae93b87f3b5c1c78',
+    grantee: BLABS_VEBAL_MULTISIG,
+    target: '0x3c1d00181ff86fbac0c3c52991fbfd11f6491d70',
+  },
+]);
+
+const feesAndTargetsPermissions: RoleData[] = flatten([
+  createRoleData(DAO_MULTISIG, ProtocolFeesCollector.output().ProtocolFeesCollector, [
+    ProtocolFeesCollector.actionId('ProtocolFeesCollector', 'setSwapFeePercentage(uint256)'),
+  ]),
+  createRoleData(GAUNTLET_FEE_SETTER, EVERYWHERE, [
+    StablePool.actionId('StablePool', 'setSwapFeePercentage(uint256)'),
+    MetaStablePool.actionId('MetaStablePool', 'setSwapFeePercentage(uint256)'),
+    StablePhantomPool.actionId('StablePhantomPool', 'setSwapFeePercentage(uint256)'),
+    WeightedPool.actionId('WeightedPool', 'setSwapFeePercentage(uint256)'),
+    WeightedPool.actionId('WeightedPool2Tokens', 'setSwapFeePercentage(uint256)'),
+  ]),
+  createRoleData(BALLERS_MULTISIG, EVERYWHERE, [
+    StablePhantomPool.actionId('StablePhantomPool', 'setTokenRateCacheDuration(address,uint256)'),
+    AaveLinearPool.actionId('AaveLinearPool', 'setSwapFeePercentage(uint256)'),
+    AaveLinearPool.actionId('AaveLinearPool', 'setTargets(uint256,uint256)'),
+  ]),
+]);
+
+export const roles: RoleData[] = flatten([
+  ...batchRelayerPermissions,
+  ...lidoRelayerPermissions,
+  ...gnosisProtocolRelayerPermissions,
+  ...protocolFeesPermissions,
+  ...veBALPermissions,
+  ...feesAndTargetsPermissions,
+]);
+
+export const granters: RoleData[] = flatten([
+  createRoleData(BLABS_OPS_MULTISIG, EVERYWHERE, [
+    SingleRecipientGauge.actionId('SingleRecipientGauge', 'checkpoint()'),
+  ]),
+]);
+export const revokers: RoleData[] = [];
+export const executeDelays: DelayData[] = [
+  { actionId: Vault.actionId('Vault', 'setAuthorizer(address)'), newDelay: 30 * DAY },
+  {
+    actionId: SmartWalletChecker.actionId('SmartWalletChecker', 'denylistAddress(address)'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: SmartWalletChecker.actionId('SmartWalletChecker', 'allowlistAddress(address)'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: VotingEscrowDelegationProxy.actionId('VotingEscrowDelegationProxy', 'setDelegation(address)'),
+    newDelay: 7 * DAY,
```

Comment:
I think this should be higher - it controls the entire L1 token distribution process

Reply:
Increased to 14 days in [9f5b54f](https://github.com/balancer-labs/balancer-v2-monorepo/pull/1453/commits/9f5b54fd8fe3e075101ffd35c1ebbdafa7efd8f8)